packages/autocomplete-client/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@meilisearch/autocomplete-client",
+  "version": "0.1.0",
+  "private": false,
+  "description": "The search client to use Meilisearch with autocomplete.js.",
+  "scripts": {
+    "clear_jest": "jest --clearCache",
+    "cleanup": "shx rm -rf dist/",
+    "test:watch": "yarn test --watchAll",
+    "test": "jest --runInBand --selectProjects dom --selectProjects node",
+    "build": "yarn cleanup && rollup -c rollup.config.js && rollup --environment NODE_ENV:production -c rollup.config.js",
+    "dev": "rollup -c rollup.config.js --watch",
+    "lint": "eslint --ext .js,.ts,.tsx .",
+    "lint:fix": "eslint --ext .js,.ts,.tsx . --fix",
+    "test:types": "yarn tsc",
+    "version": "node scripts/update_version.js"
+  },
+  "main": "./dist/autocomplete-client.umd.js",
+  "module": "./dist/autocomplete-client.esm.js",
+  "browser": "./dist/autocomplete-client.umd.js",
+  "cjs": "./dist/autocomplete-client.cjs.js",
+  "source": "src/index.ts",
+  "typings": "./dist/types/index.d.ts",
+  "types": "./dist/types/index.d.ts",
+  "sideEffects": false,
+  "bugs": {
+    "url": "https://github.com/meilisearch/instant-meilisearch/issues"
+  },
+  "files": [
+    "dist",
+    "src",
+    "templates"
+  ],
+  "keywords": [
+    "search",
+    "autocomplete",
+    "client"
+  ],
+  "author": "Meilisearch",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/meilisearch/instant-meilisearch.git"
+  },
+  "dependencies": {
+    "@meilisearch/instant-meilisearch": "*"
+  },
+  "peerDependencies": {
+    "@algolia/autocomplete-js": "^1.7.4"
+  },
+  "devDependencies": {
+    "@algolia/autocomplete-js": "^1.7.4",
+    "@babel/cli": "^7.19.3",
+    "@babel/core": "^7.20.5",
+    "@babel/preset-env": "^7.20.2",
+    "@rollup/plugin-commonjs": "^17.1.0",
+    "@rollup/plugin-node-resolve": "^11.2.0",
+    "@types/jest": "^27.0.2",
+    "babel-jest": "^27.2.2",
+    "concurrently": "^7.1.0",
+    "cssnano": "^4.1.10",
+    "eslint-config-meilisearch": "*",
+    "instantsearch.js": "^4.49.4",
```

Comment:
All the dependencies that are shared with instant-meilisearch will be moved to the root package json in another PR